Code/GraphMol/MolStandardize/TautomerCatalog/TautomerCatalogParams.h

```diff
@@ -24,6 +24,13 @@ class ROMol;
 namespace MolStandardize {
 class TautomerTransform;
 
+using TautomerTransformDefs =
```

Comment:
perhaps also #include <tuple> ?

Reply:
is this a suggestion to fix the win dll builds? That was because I forgot the magic extern thing for windows.
should be fixed now

Reply:
Yep.

Reply:
It didn't make sense to me because I didn't expect you had to export an alias, but that wasn't the issue.